index.js

```diff
@@ -4833,7 +4818,7 @@ export const createMultiXBom = async (pathList, options) => {
         listComponents(options, {}, bomData.bomJson.components, "gem", "xml")
       );
     }
-    bomData = await createCsharpBom(path, options, parentComponent);
```

Comment:
This was passing an existing parentComponent which might belong to a different type.